src/slider/Slider.ts

```diff
@@ -174,8 +182,9 @@ export default class Slider extends SliderBase<SliderProperties> {
 				})
 			]),
 			v('output', {
-				classes: this.classes(css.output),
-				for: this._inputId + ''
+				classes: this.classes(css.output, outputIsTooltip ? css.outputTooltip : null),
+				for: this._inputId + '',
```

Comment:
not a big fan of the `+ ''` approach for casting to a string. Would it not be better to use a template string? ie. 
```
for: `${this._inputId}`
```